modules/apps/transfer/keeper/grpc_query_test.go

```diff
@@ -21,10 +21,10 @@ func (suite *KeeperTestSuite) TestQueryDenomTrace() {
 		expPass  bool
 	}{
 		{
-			"invalid hex hash",
+			"invalid ibc denom",
```

Comment:
nit: could we put the success case first? :=)

Reply:
yesss